README.md

```diff
@@ -26,3 +26,21 @@ Debian build facilities are integrated into the repo at `debian/`.
 3. Install the resulting `jellyfin*.deb` file on your system.
 
 A huge thanks to Carlos Hernandez who created the Debian build configuration for Emby 3.1.1.
+
+### Windows (64 bit)
+A pre-built windows installer will be available at [The JellyFin Repository](https://repo.jellyfin.org/).
+
+1. Install the dotnet core SDK 2.1 from [Microsoft's Webpage](https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-2.1.500-windows-x64-installer)
+2. Clone Jellyfin into a directory of your choice. From that directory run in powershell `dotnet publish -c Release -r win10-x64 MediaBrowser.sln -o $Env:APPDATA\Jellyfin-Server` or in CMD `dotnet publish -c Release -r win10-x64 MediaBrowser.sln -o %APPDATA%\Jellyfin-Server`
+3. (Optional) Copy the ffmpeg binaries into the Jellyfin directory:
```

Comment:
Does that static ffmpeg build provide hardware acceleration support? Linux one (from John van Sickle) does not.

Reply:
Looks like it does. At least for NVIDIA: https://ffmpeg.zeranoe.com/forum/viewtopic.php?f=2&t=4860